### ISSUE TITLE
Jeg Får data, men det stemmer ikke overens med virkeligheden.

### ISSUE THREAD

**Issue by Guitarans** [AMajland/Danfoss-TLX#5]
Hejsa. 
Spændende projekt, og jeg var glad for at der skete noget, det er bare som om der er noget tosset.
jeg får data ind og siden fra min esp32 viser fx:

Grid current, DC L1 | -1735518720 | mA | GridDC1
Grid current, DC L2 | -851055616 | mA | GridDC2
Grid current, DC L3 | -393920256 | mA | GridDC3

og når jeg læser fra seriel, står der ofte : 3. TO BIG ud for tingene,
har du nogen ide om hvad der sker?
Mange tak..

**Comment by AMajland** [comment]
Nej, ikke rigtigt. Prøv at debugge et enkelt request (udkommentere det meste kode) og se på resultatet
